.browserslistrc

```diff
@@ -1,3 +1,3 @@
 > 1%
 last 2 versions
-not ie <= 10
+not dead
```

Comment:
This _should_ remove polyfills for IE11 and maybe other old browsers.

I saw somewhere that Vue3 is not compatible with IE11 anyway, so I think this is a good change.